### PR TITLE
refactor: change scrollbar radius and color

### PR DESCRIPTION
**Pull request by ErickSharp**
## Summary of changes
Change scrollbar color and border radius to look less out of place. 

## Screenshots(if applicable)
Before:
image

After:
image

### CODE REVIEW

File: styles/global.scss [hunk 1]
```diff
@@ -78,13 +78,13 @@ body {
 
   &::-webkit-scrollbar-thumb {
     background-color: $scrollbar-thumb-color;
-    border-radius: 20px;
+    border-radius: 1px;
```
Comment (FoxtrotSierra6829) [inline styles/global.scss:81]: Just no.

Comment (ErickSharp) [reply styles/global.scss:81]: Haha, alright fixing rn.

**Comment by GhostEagle68** [comment]
Definitely needs some type of animation when you click on it and hover over it. Way too static

**Comment by NathanInnes** [comment]
Please use HSL values instead of RGB

**Comment by FoxtrotSierra6829** [comment]
@NathanInnes don't you mean HEX? Cause HSL is really inconvenient

**Comment by NathanInnes** [comment]
Maybe not in this application but plans are to turn everywhere possible to HSL as it's more convienient to work with, if you need a variation on the colour of choice you can adjust the values in a more pratical way of thinking. Need a different colour? Use hue value, need to change how much of it there is? use saturation etc.

**Comment by FoxtrotSierra6829** [comment]
IDK, I don't like the HSL approach because it's very inconsistent. E.g. I have hsl(240,100%,50%) and I change it to hsl(180,100%,50%), it's going to emit more light, even though I didn't change the lightness parameter. In HEX I can tell if I go from #0000FF to #00FFFF it's going to be more lit because It's physically 2 times the light.

**Comment by NathanInnes** [comment]
Arguement can be made about this though, how is ```AAFF00``` consitent with ```B1E052``` when its a slight variation on saturation and lightness

**Comment by FoxtrotSierra6829** [comment]
hmm, yeah maybe preference and habituation

**Comment by ErickSharp** [comment]
all set 😁

**Comment by ErickSharp** [comment]
just realized i didnt create a branch for this PR.bruh

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**